server/src/main/java/org/elasticsearch/common/settings/LocallyMountedSecrets.java

```diff
@@ -48,24 +56,56 @@
  *                  "version": "1",
  *                  "compatibility": "8.7.0"
  *              },
- *              "secrets": {
+ *              "string_secrets": {
  *                  "secure.setting.key.one": "aaa",
  *                  "secure.setting.key.two": "bbb"
  *              }
+ *              "file_secrets": {
+ *                  "secure.setting.key.three": "Y2Nj"
+ *              }
  *         }
  */
 public class LocallyMountedSecrets implements SecureSettings {
 
     public static final String SECRETS_FILE_NAME = "secrets.json";
     public static final String SECRETS_DIRECTORY = "secrets";
 
-    public static final ParseField SECRETS_FIELD = new ParseField("secrets");
+    public static final ParseField STRING_SECRETS_FIELD = new ParseField("string_secrets", "secrets");
```

Comment:
perhaps add a TODO here to remember to remove "secrets" once CP has transition off it?